### PR TITLE
OM-806 | Mobile navigation

### PR DESCRIPTION
**Pull request by tuovinensanttu**
Hide logged in user links if user is not logged in.

### CODE REVIEW

File: src/common/fullscreenNavigation/FullscreenNavigation.tsx [hunk 1]
```diff
@@ -42,17 +42,21 @@ function FullscreenNavigation(props: Props) {
...
-          </NavLink>
-          <NavLink
-            to="/connected-services"
-            className={styles.navLink}
-            onClick={closeNav}
-          >
-            {t('nav.services')}
-          </NavLink>
           {isAuthenticated && (
+            <React.Fragment>
```
Comment (Jokinen) [inline src/common/fullscreenNavigation/FullscreenNavigation.tsx:46]: Shouldn't our version of babel support `<>`? Do you simply prefer this notation?

**Review by Jokinen — COMMENTED** [review]
Looks to be working 👍

File: src/common/fullscreenNavigation/FullscreenNavigation.tsx (hunk 1 above)
Comment (tuovinensanttu) [reply src/common/fullscreenNavigation/FullscreenNavigation.tsx:46]: I just prefer it 😄, using `Fragment` just makes it feel 'cleaner', if that makes any sense...

**Review by Jokinen — APPROVED** [review]
Sorry, I made a comment when I meant to approve :D

File: src/common/fullscreenNavigation/FullscreenNavigation.tsx (hunk 1 above)
Comment (badrange) [reply src/common/fullscreenNavigation/FullscreenNavigation.tsx:46]: I prefer <> - wonder if we're going to start a vi vs emacs war 😄

**Pull request merged**